src/themes/theme.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { Theme } from '@material-ui/core/styles';
+
```

Comment:
I think this could include a function `getHeaderLogo` which keeps some of the specific code here. This could include any information about adding additional padding to the logo rather than including facility specific code elsewhere.